src/ace_time/hw/stmrtc.h

```diff
@@ -0,0 +1,50 @@
+/*
+ * MIT License
+ * Copyright (c) 2018 Brian T. Park, Anatoli Arkhipenko
+ */
+
+#ifndef ACE_TIME_HW_STMRTC_H
+#define ACE_TIME_HW_STMRTC_H
+
+#if ! defined(UNIX_HOST_DUINO)
+#if defined(ARDUINO_ARCH_STM32)
+
+#include <stdint.h>
+#include <STM32RTC.h>
+
+namespace ace_time {
+namespace hw {
+
+class HardwareDateTime;
+
+/**
+ * A class that reads and writes HardwareDateTime and HardwareTemperature from a
+ * STM RTC chip. This class is designed to access just enough features of the
+ * RTC chip to implement the ace_time::StmRTCClock class. It is not
+ * meant to provide access to all the features of the RTC chip.
+ 
+ * Requires https://github.com/stm32duino/STM32RTC
+ */
+class STMRTC {
```

Comment:
I guess the other library uses the name STM32RTC, so you can't use that. The problem I have with "STMRTC" is that it is ALL-CAPS, so it looks like a `#define`. I think I would prefer "StmRtc", even though it looks slightly funny at first. (After a while, you get used to it...)